docs/4.x/reporting-api.md

```diff
@@ -32,7 +32,7 @@ The APIs let you programmatically request any analytics reports from Matomo, for
 *   **date**
 
     *   standard format = _YYYY-MM-DD_
-    *   magic keywords = _today_ or _yesterday_. These are relative the website timezone. For example, for a website with UTC+12 timezone, "date=today" for an API request at 5PM UTC on 2010-01-01 will return the reports for 2010-01-02.
+    *   magic keywords = _today_, _yesterday_, _lastWeek_, _lastMonth_ or _lastYear_. These are relative the website timezone. For example, for a website with UTC+12 timezone, "date=today" for an API request at 5PM UTC on 2010-01-01 will return the reports for 2010-01-02.
```

Comment:
@sgiehl can we maybe mention the new ones require Matomo 4.1?

Reply:
added a note